package/gluon-mesh-batman-adv-core/files/lib/gluon/announce/nodeinfo.d/network/mesh_interfaces

```diff
@@ -1,13 +0,0 @@
-local batman_adv = require 'gluon.batman_adv'
-
-local interfaces = {}
-
-for ifname in batman_adv.interfaces('bat0') do
-  pcall(
-    function()
-      table.insert(interfaces, util.readline(io.open('/sys/class/net/' .. ifname .. '/address')))
-    end
-  )
-end
-
-return interfaces
```

Comment:
`nodeinfo.network.mesh_interfaces` had been deprecated when `nodeinfo.network.mesh.bat0.interfaces` was added. This was before 2015.1, and the commit message mentioned that `nodeinfo.network.mesh_interfaces` would be removed after 2015.1.

At the moment, we don't have API versions or proper documentation of the API. We plan to switch to the well-documented [NetJSON](http://netjson.org/) format some time in the future (although we'd still have Gluon-specific extensions to NetJSON which would need to be documented...)